src/index.js

```diff
@@ -0,0 +1,18 @@
+import { FirstStep } from './Steps'
+class Formularz {
+    activeStep
+    wrapperHTMLElement 
+
+    constructor () {
+        this.wrapperHTMLElement = document.getElementsByTagName('body')[0]
+    }
+
+    render () {
+       this.activeStep.render() 
+    }
+}
+
+
+window.formularz = new Formularz()
```

Comment:
To powinno być przekazane jakoś przy inicjalizacji klasy. 

   ```
 constructor (formInstance) {
        this.formInstance = formInstance
    }

```
Wtedy wywołanie `formularz.render().` To zamiast tego `this.formInstance.render()` ale takie mam przypuszczenie bo nie podoba mi się ten zapis z `window`. Tak jakby stażysta zakładał, że w `window` istnieje `formularz`.